# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../geany/plugins_init.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- geany-0.14.orig/src/plugins.c	2008-04-17 17:48:12.000000000 +0000
+++ geany-0.14/src/plugins.c	2008-09-10 17:15:45.000000000 +0000
@@ -605,8 +605,8 @@
 {
 	guint i, len;
 
-	len = g_strv_length(active_plugins_pref);
-	if (active_plugins_pref == NULL || len == 0)
+	if (active_plugins_pref == NULL || 
+			(len = g_strv_length(active_plugins_pref)) == 0);
 		return;
 
 	for (i = 0; i < len; i++)
